assets/js/components/TourTooltip.js

```diff
@@ -0,0 +1,147 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Card, { CardActions } from '@material/react-card';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import CloseIcon from '../../svg/close.svg';
+import Button from './Button';
+
+/**
+ * Creates an array of length equal to size with incremental values starting at 0.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} size Array length.
+ * @return {number[]} Array of incremental numbers.
+ * @example createIncrementalArrayBySize(3); // [0, 1, 2]
+ */
+const createIncrementalArrayBySize = ( size ) => new Array( size ).fill().map( ( _, i ) => i );
+
+export default function TourTooltip( {
+	backProps,
+	closeProps,
+	index,
+	isLastStep,
+	primaryProps,
+	size,
+	step,
+	tooltipProps,
+} ) {
+	const indicatorArray = createIncrementalArrayBySize( size );
+	const getIndicatorClassName = ( indicatorIndex ) => classnames( 'googlesitekit-tooltip-indicator', {
+		active: indicatorIndex === index,
+	} );
+
+	const gotItText = __( 'Got it', 'google-site-kit' );
+
+	return (
+		<div className="googlesitekit-tour-tooltip" { ...tooltipProps }>
+			<Card className="googlesitekit-tooltip-card">
+				<div className="googlesitekit-tooltip-body">
+					<h2 className="googlesitekit-tooltip-title">
+						{ step.title }
+					</h2>
+					<div className="googlesitekit-tooltip-content">
+						{ step.content }
+					</div>
+				</div>
+				<CardActions className="googlesitekit-tooltip-actions">
+					<ul className="googlesitekit-tooltip-indicators">
+						{ indicatorArray.map( ( indicatorIndex ) => (
+							<li
+								key={ `indicator-${ indicatorIndex }` }
+								className={ getIndicatorClassName( indicatorIndex ) }
+							/>
+						) ) }
+					</ul>
+					<div className="googlesitekit-tooltip-buttons">
+						{ index !== 0 && (
+							<Button
+								className="googlesitekit-tooltip-button"
+								text
+								{ ...backProps }
+							>
+								{ __( 'Back', 'google-site-kit' ) }
+							</Button>
+						) }
+						{ ! isLastStep && (
+							<Button
+								className="googlesitekit-tooltip-button"
+								text
+								{ ...primaryProps }
+							>
+								{ __( 'Next', 'google-site-kit' ) }
+							</Button>
+						) }
+						{ isLastStep && (
+							<Button
+								className="googlesitekit-tooltip-button"
+								text
+								{ ...closeProps }
+								aria-label={ gotItText }
+								title={ gotItText }
+							>
+								{ gotItText }
+							</Button>
+						) }
+					</div>
+				</CardActions>
+				<Button
+					className="googlesitekit-tooltip-close"
+					text
+					{ ...closeProps }
+				>
+					<CloseIcon width="14" height="14" />
```

Comment:
This icon should be passed via `icon` prop:

```javascript
<Button
    className="googlesitekit-tooltip-close"
    text
    icon={ <CloseIcon width="14" height="14" /> }
    { ...closeProps }
>
```

Also currently the Button component requires a string to be passed as its children. We probably need to update the Button component to make the children prop be optional.